### ISSUE TITLE
How to get Points from the Input Cloud that lie on the Output Convex Hull Mesh?

### ISSUE THREAD

**Issue by lokitkhemka** [isl-org/Open3D#5933]
### Checklist

- [X] I have searched for [similar issues](https://github.com/isl-org/Open3D/issues).
- [X] For Python issues, I have tested with the [latest development wheel](http://www.open3d.org/docs/latest/getting_started.html#development-version-pip).
- [X] I have checked the [release documentation](http://www.open3d.org/docs/release/) and the [latest documentation](http://www.open3d.org/docs/latest/) (for `master` branch).

### My Question

The question is present here in detail:

https://stackoverflow.com/questions/75501131/points-of-the-point-cloud-lying-on-convex-hull-computed-using-open3d

Basically, all I am looking for a point cloud of points that lie on convex hull mesh. I could not find anything. Any help provided will be greatly appreciated. Thank you.
